packages/web/components/Dashboard/Post/Post.tsx

```diff
@@ -553,6 +556,23 @@ const Post = ({ post, currentUser, refetch }: IPostProps) => {
   }
 
   const activeThread = post.threads.find((thread: ThreadType) => thread.id === activeThreadId)
+
+  const handleBumpPost = () => {
+    if (post.bumpCount >= POST_BUMP_LIMIT) {
+      toast.error(t('postBumpLimitError'))
+      return
+    }
+    bumpPost()
+  }
+
+  const [bumpPost] = useBumpPostMutation({
+    variables: { postId: post.id },
+    onCompleted: () => {
+      toast.success(`${t('bumpPostSuccess')} ${POST_BUMP_LIMIT - (post.bumpCount + 1)}`)
```

Comment:
Let's use i18next's interpolation here ([docs](https://www.i18next.com/translation-function/interpolation#basic)) in case it makes sense to put the number somewhere other than the end of the string in translation.

Reply:
Great call! 🙌🏼